scripts/buildko

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export KO_LOG=info 
+export VERSION="${VERSION:-v0.1.0}"
+export GIT_COMMIT="${GIT_COMMIT:-dev}"
+export BUILD_DATE=$(date -u +%FT%TZ)
+
+# Ensure go.work file exists
+if [ ! -f go.work ]; then
+  go work init
+  go work use \
+    ./fault-quarantine-module \
+    ./fault-remediation-module \
+    ./health-events-analyzer \
+    ./health-monitors/csp-health-monitor/cmd/csp-health-monitor \
+    ./health-monitors/csp-health-monitor/cmd/maintenance-notifier \
+    ./labeler-module \
+    ./node-drainer-module \
+    ./platform-connectors 
+fi
+
+ko build -B --sbom=cyclonedx --tags="${VERSION}-slim" --tags=latest \
```

Comment:
Can we remove the push to latest? Let's only keep the slim tag for now and once we cut over to ko completely we can add latest, wdyt?